digitalstrategie/assets/scss/components/_ds-block.scss

```diff
@@ -15,55 +15,35 @@
         color: $body-bg !important;
         background-color: $green;
 
-        .accordion-button,
-        .accordion-body,
         // teaser centered styling
         .background-image + .content-container *:not(.button),
         // richtext and cta centered styling
         .button,
         h2,
         h3,
+        h3:after,
         h4 {
             background-color: $green;
-            color: $body-bg;
+            color: $body-bg !important;
```

Comment:
just saw that the lines between the FAQuestions are differently thick? can you replicate it?
And the top padding/margin is quite big.

let me know if you want to tackle it now or i add issues?

Reply:
@khamui can you send me screenshots? it was looking like the styleguide on my machine

Reply:
update: on my macbook screen it looks fine. merging and doublecheck when people testing